### ISSUE TITLE
[BUG] Lets Encrypt Auto Renewal Reverts Webmail Client back to Roundcube

### ISSUE THREAD

**Issue by johnnyq** [hestiacp/hestiacp#1980]
**Describe the bug**
I noticed today when I went to webmail. It reverted from Rainloop back to Roundcube. So I went into edit mail domain and webmail client and it stated it was rainloop even though it was roundcube so I hit save and it fixed the the issue. I also noticed the Cert got renewed July 9th so im guessing the renewal triggered webmail client to revert.

**NOTE: Do not include any personal or sensitive information, such as email addresses or passwords.**

**To Reproduce**
What steps did you take when the issue occured? 
1. Create Mail Domain
2. Install Rainloop
3. Set Rainloop as the default Webmail client
4. Wait for LE to renew on the mail domains

**Expected behavior**
It should not revert back to roundcube if rainloop is used

**Operating system:**
Ubuntu 20.04

**Hestia Control Panel version:**
1.4.5

**Comment by jaapmarcus** [comment]
I don't know if that was the issue I know their was a bug in the update script when upgrading from 1.4.3 to 1.4.4 maybe this was the issue.

Please reopen the issue If it happens again...

**Comment by johnnyq** [comment]
Hi @jaapmarcus 

I did some more testing and the issue still seems to be present. I tested on 2 Platforms my test platform and productive platform running same version of hestia 1.4.5 and Ubuntu 20.04

I manually renewed LE on the mail domain that had Rainloop webmail active and it reverted back to roundcube even though rainloop is selected for web mail client. 

I changed it to roundcube then hit save then back to rainloop then save. 

Now rainloop is back to the default.

It looks like the issue is when LE is renewed manually or automatically it reverts even though the web interface has rainloop selected.

**Comment by jaapmarcus** [comment]
https://github.com/hestiacp/hestiacp/blob/d657f5a23227f69b0f1d9ed57131057fa79867f3/bin/v-add-mail-domain-ssl#L81-L90

This might be the issue

**Comment by johnnyq** [comment]
@jaapmarcus Fantastic! that looks like the fix! Thank you for all that you and your team do =]

**Comment by johnnyq** [comment]
@jaapmarcus Today I noticed one of my domains that use roundcube webmail was inaccessible and gave them the custom 404 page. I went into the control panel and noticed Lets Encrypt renewed the Certificate so I went into Edit mail domain and Rainloop was selected for webmail even though they were using roundcube. I selected roundcube and clicked save and the webmail started working.

I did a manual Lets Encrypt renew on several other domains that have a mix of  roundcube and rainloop webmail and every time the webmail showed a 404, and the selected mail client showed rainloop. Easily fixed with selecting the webmail client and clicking save.
